releasenotes/add-verified-field-to-analysis-results-5926fa1428ef1df5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Introduced a new property (verified) on Analysis results. This property
+    identifies if the result quality was verified by a human.
```

Comment:
```suggestion
    :class:`qiskit.providers.ibmq.analysis_result.AnalysisResult` now has an additional
    `verified` attribute which identifies if the `quality` has been verified by a human.

```

Reply:
^ we use reStructuredText for doc rendering, which uses 2 backticks for inline literals and single for italics (I assume you want `verified` and not _verified_).

Reply:
Oops, yeah, keep forgetting that (I used to do everything in rst for years and now I'm too used to markdown). Updated the suggestion.